.travis.yml

```diff
@@ -1,21 +1,20 @@
-language: php
+version: ~> 1.0
 
-php: 5.6
+import:
+  - silverstripe/silverstripe-travis-shared:config/provision/cwp.yml
 
 env:
   global:
   - TRAVIS_NODE_VERSION="6"
 
-cache:
-  directories:
-    - node_modules
+jobs:
+  include:
+    - php: 7.4
```

Comment:
what's the reason for having a custom matrix here?

Reply:
There's no PHPUNIT tests.  There's virtually nothing, may make sense to add in NPM_TEST at a later date, though at this stage I don't want to add anything, simply migrate to shared configs